### ISSUE TITLE
type SRV api failed

### ISSUE THREAD

**Issue by YoSiJo** [inwx/ansible-collection#18]
I try this task:

```yaml
…
  tasks:
    - name: Manage DNS on INWX
      inwx.collection.dns:
        username: "{{ inxw_username }}"
        password: "{{ inwx_password }}"
        domain: waima.nu
        record: _mumble._tcp
        type: SRV
        value: mumble.lxc.waima.nu
        port: 64738
        priority: 10
        weight: 10
        state:    "{{ item.state    | default('present') }}"
      loop: "{{ waimanu_domains }}"
      when: waimanu_domains is defined
```
And have this responce:

```log
failed: [localhost] (
  item={'domain': 'waima.nu', 'record': '_mumble._tcp', 'type': 'SRV', 'value': 'mumble.lxc.waima.nu', 'priority': '10', 'weight': '10', 'port': '64738'}
) => {
  "ansible_loop_var": "item", "changed": false, "item": {
    "domain": "waima.nu", "port": "64738", "priority": "10", "record": "_mumble._tcp", "type": "SRV", "value": "mumble.lxc.waima.nu", "weight": "10"
  }, "msg": "API error.", "result": {
    "api_response": {
      "code": 2005, "details": [{
        "code": "PARAM_INVALID", "location": "request.content", "locationValue": "1 None mumble.lxc.waima.nu", "msg": "The parameter '' is invalid"
      }], "msg": "Parameter value syntax error"
    }
  }
}
```
I think here handle the api this wrong, also i have no idea what.

**Comment by NickUfer** [comment]
Somehow the `port` is not set correctly in the request... but it is defined. I will test this tomorrow how this is possible.

**Comment by NickUfer** [comment]
This should now be fixed with version 1.2.2. I tested it with the examples in this project and your exact configuration :+1:
